app/assets/stylesheets/govuk_publishing_components/application.scss

```diff
@@ -4,4 +4,176 @@
 @import "typography";
 @import "colours";
 
-@import "component_guide";
+$prism-background: #f5f2f0;
```

Comment:
It's not 100% clear to my why this change is required, but I can shed a little more light on the situation which may help others who know more about sass and sprockets.  If we add a variable to `application.scss` and try to reference it in `component_guide.scss` it doesn't work, so the failure isn't that we're not loading in the frontend toolkit variables properly; it's that we're not sharing any variables at all.  

If we move `component_guide.scss` into a subfolder (e.g. `/app/assets/stylesheets/govuk_publishing_components/whyyyy` and change the import statement accordingly (e.g. `@import "whyyyy/component_guide";`) then the stylesheet does render and can reference the variables imported from govuk_frontend_toolkit or added directly here.

This problem only surfaced when we used this gem in static, and not when we used it in government-frontend, so I suspect it's down to differences in the versions of gems used in the asset-pipeline: sass, sass-rails, sprockets, sprockets-rails, and rails itself.

In either case we have to make a large amount of git diff noise to move the file around so it's not clear which is better.